### ISSUE TITLE
Problem using pre-created ACM cert (Invalid index)

### ISSUE THREAD

**Issue by richardcase** [terraform-aws-modules/terraform-aws-atlantis#80]
I am running into the following error when using a pre-existing certificate:
```bash
Error: Invalid index

  on .terraform/modules/atlantis.acm/terraform-aws-modules-terraform-aws-acm-566067c/main.tf line 6, in locals:
   6:   validation_domains = [for k, v in aws_acm_certificate.this[0].domain_validation_options : tomap(v) if contains(local.distinct_domain_names, v.domain_name)]
    |----------------
    | aws_acm_certificate.this is empty tuple

The given key does not identify an element in this collection value.
```

This is an issue with the underlying ACM module, see [here](https://github.com/terraform-aws-modules/terraform-aws-acm/issues/26). However, there is a [PR open](https://github.com/terraform-aws-modules/terraform-aws-acm/pull/28) for a change to the module.

This occurs if you explicit supply a certificate ARN when using the atlantis module:
```tf
module "atlantis" {
  source  = "terraform-aws-modules/atlantis/aws"
  version = "~> 2.0"

  name = "atlantis"

  # ACM (SSL certificate) 
  certificate_arn = module.acm.this_acm_certificate_arn
...
```

**Comment by antonbabenko** [comment]
Thanks for reporting this issue!

[terraform-aws-acm](https://github.com/terraform-aws-modules/terraform-aws-acm) module has been just fixed, so I've updated this one also - [v2.3.0](https://github.com/terraform-aws-modules/terraform-aws-atlantis/releases/v2.3.0) of this module is out.

**Comment by richardcase** [comment]
Thanks @antonbabenko
